modules/nf-core/sourmash/compare/main.nf

```diff
@@ -0,0 +1,43 @@
+process SOURMASH_COMPARE {
+    tag "$meta.id"
+    label 'process_low'
```

Comment:
Since there is a `--processes` option, should this be medium or high? Would there be a benefit?

Reply:
I think for the general use case (e.g. less than 100 samples), `process_low` is appropriate. Since `sourmash compare` does and all-by-all comparison, once you have ~> 5000 samples, threading starts to help. Most people that I see using `compare` though only use it for a handful of samples. I didn't know what was best though as far as the `process*` label goes --  is it better to drastically overestimate so it fits every ones use case, or go by what 98% of people will use it for?

Reply:
It's better to have the default be low. You might even apply `process_single`. Users can always configure this themselves.

Reply:
ok thank you for the suggestion! I'll update to `process_single`